lib/ansible/config/base.yml

```diff
@@ -1845,7 +1845,7 @@ SHOW_CUSTOM_STATS:
   type: bool
 STRING_TYPE_FILTERS:
   name: Filters to preserve strings
-  default: [string, to_json, to_nice_json, to_yaml, ppretty, json]
+  default: [string, to_json, to_nice_json, to_yaml, to_nice_yaml, ppretty, json]
```

Comment:
in the future (not blocking this PR) should we instead add a property to filters to let them declare expected output type?